test/functional/rpc_psbt.py

```
@@ -9,15 +9,15 @@
 from itertools import product
 
 from test_framework.descriptors import descsum_create
-from test_framework.key import ECKey
+from test_framework.key import H_POINT
```

Ruff (0.12.2)

`test_framework.key.H_POINT` imported but unused

Remove unused import: `test_framework.key.H_POINT`

(F401)